doc/pod1/cellcc_config.pod

```diff
@@ -103,12 +103,40 @@ mysql database called "cellcc" on host db.example.com using port 3306.
 
 The username and password to use when connecting to the B<db/ro/dsn> database.
 
+=item B<db/ro/options>
+
+This optional directive can be used to add / alter the settings to be used when
+contacting the database. For example, the admin can specify the timeout to use
+when reading or connecting to the B<db/ro/dsn> database. For the MySQL driver,
+the following timeout options are available:
```

Comment:
any DBI option is fine; we don't need to list these. but we can/should say what our default is, which is these three directives set to 5 minutes